quesma/clickhouse/alter_table_test.go

```diff
@@ -0,0 +1,54 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+package clickhouse
+
+import (
+	"github.com/stretchr/testify/assert"
+	"quesma/concurrent"
+	"quesma/quesma/config"
+	"quesma/quesma/types"
+	"testing"
+)
+
+func TestAlterTable(t *testing.T) {
+	chConfig := &ChTableConfig{
+		hasTimestamp:         true,
+		timestampDefaultsNow: true,
+		engine:               "MergeTree",
+		orderBy:              "(timestamp)",
+		partitionBy:          "",
+		primaryKey:           "",
+		ttl:                  "",
+		attributes: []Attribute{
+			NewDefaultStringAttribute(),
+		},
+		castUnsupportedAttrValueTypesToString: true,
+		preferCastingToOthers:                 true,
```

Comment:
Unrelated to this PR, but since `others` were removed, `preferCastingToOthers` should also be removed?

Reply:
Yes, true. We can do this cleanup as followup